op-challenger/game/fault/register.go

```diff
@@ -90,7 +89,10 @@ func registerOutputAlphabet(
 			if err != nil {
 				return nil, err
 			}
-			accessor, err := outputs.NewOutputAlphabetTraceAccessor(ctx, logger, m, cfg, gameDepth, splitDepth, prestateBlock, poststateBlock)
+			validateAbsolutePrestate := func(ctx context.Context, provider faultTypes.TraceProvider) error {
+				return ValidateAbsolutePrestate(ctx, provider, contract.GetGenesisOutputRoot)
+			}
```

Comment:
Admittedly verifying the VM absolute pre-state is a bit of a pain because you have to create the `TraceProvider` to get it and that doesn't normally happen here.  Maybe it's worth adding that in a separate PR since it will likely take a bit of refactoring to achieve.

We definitely want to validate the VM prestate upfront though as its easy to misconfigure and would mean you could lose bonds.